client/src/ui/BitbakeCommands.ts

```diff
@@ -320,16 +320,30 @@ async function devtoolModifyCommand (bitbakeWorkspace: BitbakeWorkspace, bitBake
   }
 }
 
-async function devtoolSDKFallbackCommand (bitbakeWorkspace: BitbakeWorkspace, bitBakeProjectScanner: BitBakeProjectScanner, uri?: any): Promise<void> {
+async function devtoolSDKFallbackCommand (bitbakeWorkspace: BitbakeWorkspace, bitBakeProjectScanner: BitBakeProjectScanner, languageClient: LanguageClient, uri?: any): Promise<void> {
   const chosenRecipe = await selectRecipe(bitbakeWorkspace, bitBakeProjectScanner, uri)
   if (chosenRecipe !== undefined) {
     logger.debug(`Command: devtool-sdk-fallback: ${chosenRecipe}`)
     const workspace = bitBakeProjectScanner.scanResult._workspaces.find((workspace) => workspace.name === chosenRecipe)
     if (workspace === undefined) throw new Error('Devtool Workspace not found')
-    configureDevtoolSDKFallback(workspace, bitBakeProjectScanner.bitbakeDriver.bitbakeSettings, bitBakeProjectScanner.bitbakeDriver.activeBuildConfiguration)
+    const resolvedWorkspace: DevtoolWorkspaceInfo = {
+      name: workspace.name,
+      path: await bitBakeProjectScanner.resolveContainerPath(workspace.path) ?? workspace.path
+    }
+    configureDevtoolSDKFallback(resolvedWorkspace, bitBakeProjectScanner.bitbakeDriver.bitbakeSettings, bitBakeProjectScanner.bitbakeDriver.activeBuildConfiguration)
+    await generateCPPProperties(resolvedWorkspace, bitBakeProjectScanner, languageClient)
+    showSDKConfigurationDone(chosenRecipe)
   }
 }
 
+function showSDKConfigurationDone (recipe: string): void {
+  void vscode.window.showInformationMessage(`Devtool workspace for ${recipe} successfully configured`, 'Open Workspace').then((choice) => {
+    if (choice === 'Open Workspace') {
+      void vscode.commands.executeCommand('bitbake.devtool-open-workspace', recipe)
```

Comment:
so we don't care about the outcome of this async command by putting the `void`, do we want to handle it when it fails to open?

Reply:
No, we have no backup plan and VSCode will show an error on its own. I don't think we need to catch this error.